multichain-testing/test/support.ts

```diff
@@ -79,13 +79,13 @@ export const commonSetup = async (t: ExecutionContext) => {
     console.error('setupRegistry failed', e);
     throw e;
   }
-  const tools = await makeAgdTools(t.log, childProcess);
-  const keyring = await makeKeyring(tools);
-  const deployBuilder = makeDeployBuilder(tools, fse.readJSON, execa);
   const retryUntilCondition = makeRetryUntilCondition({
     log: t.log,
     setTimeout: globalThis.setTimeout,
   });
+  const tools = await makeAgdTools(t.log, childProcess, retryUntilCondition);
```

Comment:
though... now `tools` and everything derived from it carries the ambient authority along.

`childProcess` was already ambient authority though.